gwcs/wcs.py

```diff
@@ -60,6 +104,7 @@ def __init__(self, forward_transform=None, input_frame='detector', output_frame=
         self._initialize_wcs(forward_transform, input_frame, output_frame)
         self._pixel_shape = None
         self._pipeline = [Step(*step) for step in self._pipeline]
+        self._calc_approx_inv(max_inv_pix_error=5, inv_degree=None)
```

Comment:
I would say it should **not** be settable by the user unless you believe there could be images with transformations so mis-behaved that an error of 5 pixels for the initial guess would be too large for the iterative process to converge.

Reply:
There may be reasons that a different number may work better. E.g., they can accept a larger error and still get good convergence leading to faster results. Why not let them access that?

Reply:
> E.g., they can accept a larger error and still get good convergence leading to faster results.

Under what circumstances a large error in initial estimate would result in faster convergence. I can't see it.

The only benefit in allowing large error here is to speed-up computation of the _polynomial for_ initial approximation of the solution. I think we can just forget that. For HST, discrepancy between `wcs_world2pix` and the "true" solution, as I remember, could be tens of pixels (if not hundreds at the edges or a little bit outside of the image), and the method was converging pretty fast as I remember (3-5 iterations). In order to detect divergence one needs at least 2 iterations (corrections) and so there is not much to save here, I think. 

I general I am for giving users as much control as possible but I think this is one that does not matter much and only complicates an already complicated function. Moreover, even for FITS WCS users do not have control over the accuracy of the initial approximation (because instead of fitting polynomials I used the analytic inverse for non-distorted transformation).